src/azul/service/hca_response_v5.py

```diff
@@ -285,11 +258,11 @@ def make_protocols(self, entry):
         protocols = []
         for protocol in entry["contents"]["protocols"]:
             translated_process = {
-                "libraryConstructionApproach": protocol.get("library_construction_approach", []),
-                "instrumentManufacturerModel": protocol.get("instrument_manufacturer_model", []),
-                "pairedEnd": protocol.get("paired_end", []),
-                "workflow": protocol.get("workflow", []),
-                "assayType": protocol.get("assay_type", []),
+                "libraryConstructionApproach": self.sorted(protocol.get("library_construction_approach", [])),
```

Comment:
Why are we sorting the response again? Shouldn't the document fields already be sorted?